### ISSUE TITLE
[Feature Request] Support only note admonitor mode

### ISSUE THREAD

**Issue by Quorafind** [tadashi-aikawa/obsidian-old-note-admonitor#3]
I found it is so uesful for me when I set it as a always highlight message ( not only in markdown view, but also other views).

![image](https://user-images.githubusercontent.com/13215013/212638254-bbfc83db-0fe2-4332-97c2-ea341ffb3f61.png)

I did it in fork, but I think it would be better to have this feature in the plugin itself. If you think this is good, but you dont have time to add this feature, I can make some changes to this.

**Comment by tadashi-aikawa** [comment]
Hi, @Quorafind :)

> not only in markdown view, but also other views

Does "other views" means the files except for markdown?

**Comment by Quorafind** [comment]
Yes, and include Graph view, or other view in the mod-root(means that views aren't in left sidebar or right sidebar)

**Comment by tadashi-aikawa** [comment]
The target of this plugin is a file like a markdown. So it should not show the warning in the Graph view.
